udata/core/user/commands.py

```diff
@@ -37,6 +37,7 @@ def create():
         data['password'] = encrypt_password(data['password'])
         del data['password_confirm']
         user = datastore.create_user(**data)
+        user.confirmed_at = datetime.utcnow()
```

Comment:
It already is: https://github.com/etalab/udata/blob/3c2aa6ace82da212b30de6012174e866c3df312c/udata/core/user/models.py#L67